core/src/test/java/org/infinispan/partitionhandling/NumOwnersNodeCrashInSequenceTest.java

```diff
@@ -50,6 +53,9 @@ protected void createCacheManagers() throws Throwable {
       configBuilder = getDefaultClusteredCacheConfig(CacheMode.DIST_SYNC);
       configBuilder.clustering().partitionHandling().whenSplit(PartitionHandling.DENY_READ_WRITES);
       configBuilder.clustering().hash().numSegments(4).stateTransfer().timeout(30000);
+      globalBuilder = GlobalConfigurationBuilder.defaultClusteredBuilder();
+      // Must explicitly add with id >= 2500 so that the user marshaller is able to store it
+      globalBuilder.serialization().addAdvancedExternalizer(2500, new JGroupsAddress.Externalizer());
```

Comment:
Nope, the ID specified in the config always takes precedent over the value returned by `getId`

Reply:
Is it because the id from `Externalizer.getId()` is added to the internal externalizers map and the user marshaller only sees the externalizers map? And why >= 2500?

Reply:
`> 2500` because we have already reserved up until `2149` according to the docs and in fact we have actually gone higher when you look at JCache Ids.

https://infinispan.org/docs/stable/user_guide/user_guide.html#preassigned_externalizer_id_ranges

Reply:
The precedent of utilising the configured id vs `getId` is already established, I haven't changed anything: https://github.com/infinispan/infinispan/blob/4d1800b1232c17aa63b6a34acdc34865ff2af5c8/core/src/main/java/org/infinispan/marshall/core/ExternalExternalizers.java#L36-L38